great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -372,28 +372,21 @@ def _get_sqlalchemy_key_pair_auth_url(
             create_engine_kwargs,
         )
 
-    def get_compute_domain(
+    def get_domain_records(
         self,
         domain_kwargs: Dict,
-        domain_type: Union[str, MetricDomainTypes],
-        accessor_keys: Optional[Iterable[str]] = None,
-    ) -> Tuple[Select, dict, dict]:
-        """Uses a given batch dictionary and domain kwargs to obtain a SqlAlchemy column object.
+    ) -> Select:
```

Comment:
Selectable?

Reply:
@jcampbell I changed this type to `Selectable` consistently throughout the `SqlAlchemyExecutionEngine` class/module.  Thanks.